test/default-and-fips.cnf

```diff
@@ -0,0 +1,13 @@
+openssl_conf = openssl_init
+
+.include ../fipsinstall.conf
```

Comment:
I suspect that these need to be generated.  What you can do is this:

-   Add the following line in `test/build.info`:

    ```
    DEPEND[]=default-and-fips.cnf fips.cnf
    SOURCE[default-and-fips.cnf]=default-and-fips.cnf.in
    SOURCE[fips.cnf]=fips.cnf.in
    ```
-   rename those two .cnf files to .cnf.in, and change the .include like this:

    ``` perl
    .include {- use Cwd 'abs_path'; use File::Spec::Functions;
                abs_path(catfile($config{builddir}, 'test', 'fipsinstall.conf')); -}
    ```

Reply:
I have gone with using an environment variable instead -  I think this makes it a bit more flexible for installation.

Reply:
We could add a API also if required at a later date..